data_scraper.py

```diff
@@ -35,6 +46,14 @@ def _ensure_datetime(self, df):
             df[c] = pd.to_datetime(df[c], infer_datetime_format=True, format=self.date_format)
         return df
 
+    def write_to_s3(self, s3_client, bucket=BUCKET_NAME_LANDING_ZONE):
+        df = self.get_data()
+        csv_string = df.to_csv(sep=",", index=False)
+        timestamp = datetime.now().strftime("%Y-%m-%dT%H%M%S")
```

Comment:
Could just use `datetime.isoformat()`
```suggestion
        timestamp = datetime.now().isoformat()
```